renderdoc/driver/ihv/amd/amd_counters.h

```diff
@@ -25,20 +25,29 @@
 #pragma once
 
 #include <vector>
+#include <map>
 #include "api/replay/renderdoc_replay.h"
+#include "api/replay/data_types.h"
```

Comment:
There shouldn't be a need to directly include `data_types.h` here - it's included under `renderdoc_replay.h`. Generally external code should only ever include the root `renderdoc_replay.h` and not any of the children since there are some sensitive ordering issues and declarations may move around.

Reply:
Fixed